fnet/src/tests/info/info.cpp

```diff
@@ -69,19 +69,31 @@ TEST("info") {
 
 TEST("size of important objects")
 {
+#ifdef __APPLE__
+    EXPECT_EQUAL(192u, sizeof(FNET_IOComponent));
+#else
```

Comment:
In order not to pollute the code with multiple identical ifdef constructs I suggest adding a MUTEX_SIZE with an a single inside and use MUTEX_SIZE+N where necessary.